docs/configuration_requirements.rst

```diff
@@ -0,0 +1,19 @@
+Configuration requirements
+**************************
+
+.. only:: iso
+
+    This page shows the configuration requirements for the Inmanta Service Orchestrator |version_major| version.
+    These constraints are defined for the whole lifetime of this major version i.e.
+    they are not pinned to a specific three-digit version.
+
+
+    .. datatemplate:json:: required_configuration.json
+       :template: iso_config_requirements.tmpl
+
+.. only:: oss
+
+    This page shows the configuration requirements for version |version_major| of Inmanta.
```

Comment:
For OSS the major digit doesn't really have a special semantic meaning. I think we should use the full version here.